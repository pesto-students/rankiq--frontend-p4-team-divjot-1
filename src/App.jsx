import Layout from './components/Layout/Layout';
import './index.css';

function App() {
  return <Layout />;
}

function WrappedApp() {
  return <App />;
}

export default WrappedApp;
