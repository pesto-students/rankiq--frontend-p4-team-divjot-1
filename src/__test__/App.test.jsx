import { screen, render } from '@testing-library/react';
import { describe, it, expect } from 'vitest';
import App from '../App';

describe('App.jsx', () => {
  it('shoudl render without any errors', () => {
    const tree = render(<App />);
    expect(tree.asFragment()).toMatchSnapshot();
  });

  it('should render hello world', () => {
    render(<App />);
    expect(screen.queryByText('Rankiq hello')).toBeInTheDocument();
  });
});
