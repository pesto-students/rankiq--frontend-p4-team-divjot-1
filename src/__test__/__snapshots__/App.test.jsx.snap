// Vitest Snapshot v1

exports[`App.jsx > shoudl render without any errors 1`] = `
<DocumentFragment>
  <div>
    Rankiq hello
  </div>
</DocumentFragment>
`;
